drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -8,634 +8,651 @@
-    hud_enum HudUtility.when_occurreds
-  end
-
-  class ContactLocation < Types::BaseEnum
-    description '4.12.2'
-    graphql_name 'ContactLocation'
-    hud_enum HudUtility.contact_locations
+    hud_enum HudUtility2024.when_occurreds
   end
 
   class BedNight < Types::BaseEnum
     description '4.14'
     graphql_name 'BedNight'
-    hud_enum HudUtility.bed_night_options
+    hud_enum HudUtility2024.bed_night_options
   end
 
   class RHYServices < Types::BaseEnum
-    description '4.14.B'
+    description 'R14.2'
     graphql_name 'RHYServices'
-    hud_enum HudUtility.rhy_services_options
+    hud_enum HudUtility2024.rhy_services_options
   end
 
   class SSVFFinancialAssistance < Types::BaseEnum
-    description '4.15.B'
+    description 'V3.3'
     graphql_name 'SSVFFinancialAssistance'
-    hud_enum HudUtility.ssvf_financial_assistance_options
+    hud_enum HudUtility2024.ssvf_financial_assistance_options
   end
 
   class HousingAssessmentDisposition < Types::BaseEnum
     description '4.18.1'
     graphql_name 'HousingAssessmentDisposition'
-    hud_enum HudUtility.housing_assessment_dispositions
+    hud_enum HudUtility2024.housing_assessment_dispositions
   end
 
   class AssessmentType < Types::BaseEnum
     description '4.19.3'
     graphql_name 'AssessmentType'
-    hud_enum HudUtility.assessment_types
+    hud_enum HudUtility2024.assessment_types
   end
 
   class AssessmentLevel < Types::BaseEnum
     description '4.19.4'
     graphql_name 'AssessmentLevel'
-    hud_enum HudUtility.assessment_levels
+    hud_enum HudUtility2024.assessment_levels
   end
 
   class PrioritizationStatus < Types::BaseEnum
     description '4.19.7'
     graphql_name 'PrioritizationStatus'
-    hud_enum HudUtility.prioritization_statuses
+    hud_enum HudUtility2024.prioritization_statuses
   end
 
   class EventType < Types::BaseEnum
     description '4.20.2'
     graphql_name 'EventType'
-    hud_enum HudUtility.events
+    hud_enum HudUtility2024.events
   end
 
   class ReferralResult < Types::BaseEnum
     description '4.20.D'
     graphql_name 'ReferralResult'
-    hud_enum HudUtility.referral_results
+    hud_enum HudUtility2024.referral_results
+  end
+
+  class PreventionOutcome < Types::BaseEnum
+    description '4.21.A'
+    graphql_name 'PreventionOutcome'
+    hud_enum HudUtility2024.prevention_outcomes
+  end
+
+  class ShelterOutcome < Types::BaseEnum
+    description '4.21.E'
+    graphql_name 'ShelterOutcome'
+    hud_enum HudUtility2024.shelter_outcomes
+  end
+
+  class HousingOutcome < Types::BaseEnum
+    description '4.21.I'
+    graphql_name 'HousingOutcome'
+    hud_enum HudUtility2024.housing_outcomes
+  end
+
+  class ServiceProvided < Types::BaseEnum
+    description '4.21.6'
+    graphql_name 'ServiceProvided'
+    hud_enum HudUtility2024.service_provideds
+  end
+
+  class ServiceOutcome < Types::BaseEnum
+    description '4.21.M'
+    graphql_name 'ServiceOutcome'
+    hud_enum HudUtility2024.service_outcomes
```

Comment:
I believe all of 4.21 has been removed in the most-recent version of the spec. I can't seem to find these in the latest version of the CSV spec or data dictionary

Reply:
Great, I will remove them

Reply:
Oh, these were part of the 1.0 version of FY2024. I will still remove them since they're not used.